care/facility/models/patient_icmr.py

```diff
@@ -1,5 +1,5 @@
 import datetime
-
+from dateutil.relativedelta import *
```

Comment:
Codacy found an issue: [Unable to import 'dateutil.relativedelta'](https://app.codacy.com/gh/coronasafe/care/pullRequest?prid=5722747)